docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '2'
 
 services:
   tao:
-    image: alroniks/tao
+    image: locksmithdon/tao
```

Comment:
Why here another image? Could you show changes in Dockerfile?

Reply:
Oh, I just wasn't sure if you were going to accept this PR so I updated the image name in a subsequent commit. I wouldn't expect you to accept that change. Sorry, I should have done this in a different branch and not my `master`. Thanks for accepting the PR and changing the Docker image back to yours.